index.bs

```diff
@@ -135,7 +135,24 @@ Issue: Monkeypatch `window.open()`.
 
 # Fetch monkeypatches # {#fetch-monkeypatches}
 
-Issue: Specify monkeypatches for source/trigger registration.
+<pre class="idl">
+enum AttributionReportingEligibility {
+  "none",
+  "source",
+  "trigger",
+  "source-and-trigger",
```

Comment:
The existence of `source-and-trigger` makes me feel like an enum is not quite the right data structure to use here. WDYT about a dictionary (bikeshedding welcome):
```
fetch(<url>, {
  attributionReporting: {
    sourceEligibility: 'event',
    triggerEligibility: true,
  }
}
```
This sort of follows the advice here:
https://www.w3.org/TR/design-principles/#string-constants

"If you need to express a state which is a combination of properties, which might be expressed as a bitmask in another language, use a dictionary object instead. This object can be passed around as easily as a single bitmask value."

An added benefit is that an ARA-specific dictionary allows easier extensibility in the future.

Reply:
I'm fine with that, though I'd prefer to use a boolean for `sourceEligibility` and call it `eventSourceEligible`. Otherwise, we have to deal with values other than `event`.

Reply:
Perfect